### PR TITLE
Add getter for color property to ol.style.Icon

### LINKED ISSUE #6131: Addition getter for Color property on ol.style.Icon
I'd like to add to [ol.style.Icon](http://openlayers.org/en/latest/apidoc/ol.style.Icon.html) getter for property "color". Please add getColor() method to ol.style.Icon similar like in [ol.style.Fill](http://openlayers.org/en/latest/apidoc/ol.style.Fill.html) and [Stroke](http://openlayers.org/en/latest/apidoc/ol.style.Stroke.html) ;)
Thanks Tom

### PR DESCRIPTION
**Pull request by tohu12**
Added getColor() method to class ol.style.Icon similar like in ol.style.Fill and ol.style.Stroke

Fixes #6131.

### CODE REVIEW

File: src/ol/style/icon.js [hunk 1]
```diff
@@ -260,6 +260,16 @@ ol.style.Icon.prototype.getAnchor = function() {
 
 
 /**
+ * Get the icon color.
+ * @return {ol.Color|string} Color.
```
Comment (tchandelle) [inline src/ol/style/icon.js]: `color_` is parsed as ol.Color. So it shouldn't be a `string`

File: src/ol/style/icon.js [hunk 2]
```diff
@@ -260,6 +260,16 @@ ol.style.Icon.prototype.getAnchor = function() {
...
+ * @api
+ */
+ol.style.Icon.prototype.getColor = function() {
+  return this.color_;
```
Comment (tchandelle) [inline src/ol/style/icon.js]: and since `ol.Color` is an array, you should probably make a copy of it

Comment (tohu12) [inline src/ol/style/icon.js]: but is it same as in[ fill.js](/openlayers/ol3/blob/v3.19.1/src/ol/style/fill.js#L51) and [stroke.js](/openlayers/ol3/blob/v3.19.1/src/ol/style/stroke.js#L88) ?

Comment (tchandelle) [inline src/ol/style/icon.js]: Apparently, it's not initialised the same way in the constructor : 
[ol/style/fill.js#L19-L23](https://github.com/openlayers/ol3/blob/48c1d09a95a11e974d05582e5d4c15c5492c0254/src/ol/style/fill.js#L19-L23)
[ol/style/icon.js#L98-L103](https://github.com/openlayers/ol3/blob/48c1d09a95a11e974d05582e5d4c15c5492c0254/src/ol/style/icon.js#L98-L103)

Comment (tohu12) [inline src/ol/style/icon.js]: OK, thanks

File: src/ol/style/icon.js [hunk 3]
```diff
@@ -260,6 +260,16 @@ ol.style.Icon.prototype.getAnchor = function() {
 
 
 /**
+ * Get the icon color.
+ * @return {ol.Color} Color.
+ * @api
+ */
+ol.style.Icon.prototype.getColor = function() {
+  return this.color_.slice();
```
Comment (tchandelle) [inline src/ol/style/icon.js]: `this.color_` could be null

File: src/ol/style/icon.js [hunk 4]
```diff
@@ -260,6 +260,16 @@ ol.style.Icon.prototype.getAnchor = function() {
 
 
 /**
+ * Get the icon color.
+ * @return {ol.Color} Color.
+ * @api
+ */
+ol.style.Icon.prototype.getColor = function() {
+  return (this.color_ && this.color_.slice) ? this.color_.slice() : this.color_;
```
Comment (ahocevar) [inline src/ol/style/icon.js]: Usually our simple getters just return what was provided. So in this case, you would just always return `this.color_`.

**Comment by ahocevar** [comment]
@tohu12 Can you please sign a [CLA](https://github.com/openlayers/cla) so we can merge this? Thanks.

**Comment by tohu12** [comment]
I made "squash" over SourceTree my desktop aplication, I never do this before. I hope that now is right?

**Comment by ahocevar** [comment]
Sorry, that did not look right. I took the liberty to fix it for you.

**Comment by tohu12** [comment]
Thanks, and now ?

**Comment by tohu12** [comment]
@ahocevar I submited a digital Individual CLA

**Pull request merged**